src/compiler/python_generator.h

```diff
@@ -35,8 +35,18 @@
 #define GRPC_INTERNAL_COMPILER_PYTHON_GENERATOR_H
 
 #include <utility>
+#include <memory>
+#include <vector>
 
-#include "src/compiler/config.h"
+#ifndef GRPC_CUSTOM_STRING
```

Comment:
I didn't include `src/compiler/config.h` in `python_generator.h` and wanted the `grpc::string` functionality.